src/MarkdownTextInput.web.tsx

```diff
@@ -275,7 +275,7 @@ const MarkdownTextInput = React.forwardRef<MarkdownTextInput, MarkdownTextInputP
     );
 
     const handleOnSelect = useCallback(
-      (e) => {
+      (e: React.SyntheticEvent<HTMLDivElement>) => {
```

Comment:
Why do we have to make this change? What's the reason for the typecheck to pass then and fail now?

Reply:
We had to bump some types:
`"@types/jest": "^28.1.2",
    "@types/react": "~17.0.21",
    "@types/react-native": "0.70.0",`, because of conflicting peer dependency while installing packages for example.
    This bump is forcing us to add these `React.SyntheticEvent<HTMLDivElement>`